cpp/src/parquet/column_writer.h

```diff
@@ -131,14 +134,23 @@ class PARQUET_EXPORT ColumnWriter {
   /// \brief The number of rows written so far
   virtual int64_t rows_written() const = 0;
 
-  /// \brief The total size of the compressed pages + page headers. Some values
-  /// might be still buffered and not written to a page yet
+  /// \brief The total size of the compressed pages + page headers. Values
+  /// are still buffered and not written to a pager yet
```

Comment:
Is this the number of bytes that are buffered but not yet written?

Reply:
No, it's a bit hard to understand, maybe you can take a look at https://github.com/apache/arrow/pull/33897#issuecomment-1440131022